bin/crucible-run.json.example

```diff
@@ -0,0 +1,31 @@
+{
+    "mv-params": {
+        "global-options": [
+            {
+                "name": "common-params",
+                "params": [
+                    { "arg": "rw", "vals": [ "read", "write" ] }
+                ]
+            }
+        ],
+        "sets": [
+            {
+                "enabled": "yes",
+                "include": "common-params",
+                "params": [
+                    { "arg": "ioengine", "vals": [ "sync" ] }
+                ]
+            },
+            {
+                "include": "common-params",
+                "params": [
+                    { "arg": "bs", "vals": [ "4K", "16K" ] }
+                ]
+            }
+        ]
+    },
+    "tool-params": {
+    },
+    "passthru-args": ""
+}
+
```

Comment:
The `bin` directory seems like an odd location for this file.

Reply:
ack